### PR TITLE
Update Get-AzRecoveryServicesBackupContainer.md, Get-AzRecoveryServicesVault.md and Wait-AzRecoveryServicesBackupJob.md

### PR DESCRIPTION
**Pull request by arestarh**
## Description

This change is updating documenataion for Get-AzRecoveryServicesBackupContainer.md, Get-AzRecoveryServicesVault.md and Wait-AzRecoveryServicesBackupJob.md cmdlets and fixing formatting issues.

## Checklist

- [x ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x ] The title of the PR is clear and informative
- [x ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesVault.md [hunk 1]
```diff
@@ -19,34 +20,39 @@ Get-AzRecoveryServicesVault [-ResourceGroupName <String>] [-Name <String>]
...
 The **Get-AzRecoveryServicesVault** cmdlet gets a list of Recovery Services vaults in the current subscription.
 
 ## EXAMPLES
 
 ### Example 1
-```
-PS C:\> Get-AzRecoveryServicesVault
+
+```powershell
+Get-AzRecoveryServicesVault
```
Comment (markcowl) [inline src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesVault.md]: examples should have the powershell prompt `PS C:\>`

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md [hunk 2]
```diff
@@ -20,32 +21,41 @@ Get-AzRecoveryServicesBackupContainer [-ContainerType] <ContainerType> [[-Backup
 ```
 
 ## DESCRIPTION
+
 The **Get-AzRecoveryServicesBackupContainer** cmdlet gets a backup container.
 A Backup container encapsulates data sources that are modelled as backup items.
-Set the vault context by using the Set-AzRecoveryServicesVaultContext cmdlet before you use the current cmdlet.
+Set the vault context by using the **Set-AzRecoveryServicesVaultContext** cmdlet before you use the current cmdlet.
```
Comment (markcowl) [inline src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md]: SHouldn't we just tell them to use the VaultId parameter to specify the vault in the cmdlet?  Seems like we shouldn't mention the Set-ValultContext cmdlet if it is being deprecated.

File: src/RecoveryServices/RecoveryServices/help/Wait-AzRecoveryServicesBackupJob.md [hunk 3]
```diff
@@ -19,37 +20,31 @@ Wait-AzRecoveryServicesBackupJob [-Job] <Object> [[-Timeout] <Int64>] [-VaultId
 ```
 
 ## DESCRIPTION
+
 The **Wait-AzRecoveryServicesBackupJob** cmdlet waits for an Azure Backup job to finish.
 Backup jobs can take a long time.
 If you run a backup job as part of a script, you may want to force the script to wait for job to finish before it continues to other tasks.
 A script that includes this cmdlet can be simpler than one that polls the Backup service for the job status.
-Set the vault context by using the Set-AzRecoveryServicesVaultContext cmdlet before you use the current cmdlet.
+Set the vault context by using the **Set-AzRecoveryServicesVaultContext** cmdlet before you use the current cmdlet.
```
Comment (markcowl) [inline src/RecoveryServices/RecoveryServices/help/Wait-AzRecoveryServicesBackupJob.md]: same comment

File: src/RecoveryServices/RecoveryServices/help/Wait-AzRecoveryServicesBackupJob.md [hunk 4]
```diff
@@ -19,37 +20,31 @@ Wait-AzRecoveryServicesBackupJob [-Job] <Object> [[-Timeout] <Int64>] [-VaultId
...
-       $Job = Get-AzBackAzureRmRecoveryServicesBackupJob -Job $Job
-    }
-   Write-Host "Done!"
-    Waiting for completion... 
-    Waiting for completion... 
-    Waiting for completion... 
-    Done!
+
+```powershell
+$vault = Get-AzRecoveryServicesVault -ResourceGroupName "resourceGroup" -Name "vaultName"
```
Comment (markcowl) [inline src/RecoveryServices/RecoveryServices/help/Wait-AzRecoveryServicesBackupJob.md]: ps prompt

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md [hunk 5]
```diff
@@ -20,32 +21,41 @@ Get-AzRecoveryServicesBackupContainer [-ContainerType] <ContainerType> [[-Backup
...
+Warning: **Set-AzRecoveryServicesVaultContext** cmdlet is being deprecated in a future breaking change release. There will be no replacement for it. Please use the -VaultId parameter instead.
 
 ## EXAMPLES
 
 ### Example 1: Get a specific container
-```
-PS C:\>Get-AzRecoveryServicesContainer -ContainerType "AzureVM" -Status "Registered" -Name "V2VM";
+
+```powershell
+$vault = Get-AzRecoveryServicesVault -ResourceGroupName "resourceGroup" -Name "vaultName"
```
Comment (markcowl) [inline src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md]: ps prompt `PS C:\>`

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md [hunk 6]
```diff
@@ -20,32 +21,41 @@ Get-AzRecoveryServicesBackupContainer [-ContainerType] <ContainerType> [[-Backup
...
 ```
 
 This command gets the container named V2VM of type AzureVM.
 
 ### Example 2: Get all containers of a specific type
-```
-PS C:\>Get-AzRecoveryServicesBackupContainer -ContainerType Windows -BackupManagementType MARS
+
+```powershell
+$vault = Get-AzRecoveryServicesVault -ResourceGroupName "resourceGroup" -Name "vaultName"
```
Comment (markcowl) [inline src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md]: ps prompt

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesVault.md (hunk 1 above)
Comment (arestarh) [reply src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesVault.md]: powershell prompt **PS C:\>** was added back.

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md (hunk 2 above)
Comment (arestarh) [reply src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md]: I agree. I've changed note regarding vault context and mentioned VaultId parameter there.

File: src/RecoveryServices/RecoveryServices/help/Wait-AzRecoveryServicesBackupJob.md (hunk 4 above)
Comment (arestarh) [reply src/RecoveryServices/RecoveryServices/help/Wait-AzRecoveryServicesBackupJob.md]: PS prompt was added back.

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md (hunk 5 above)
Comment (arestarh) [reply src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md]: PS prompt was added back.

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md (hunk 6 above)
Comment (arestarh) [reply src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md]: PS prompt was added back.

File: src/RecoveryServices/RecoveryServices/help/Wait-AzRecoveryServicesBackupJob.md (hunk 3 above)
Comment (arestarh) [reply src/RecoveryServices/RecoveryServices/help/Wait-AzRecoveryServicesBackupJob.md]: agree. I've changed note regarding vault context and mentioned VaultId parameter there.

**Comment by arestarh** [comment]
@markcowl I've fixed all the stuff that you've mentioned. Thanks.
I'll fix the same ones in **Get-AzRecoveryServicesBackupJob.md** **and Set-AzRecoveryServicesVaultContext.md** that I've already created PR for.
Also, I'll try to fix another batch of cmdlet's documentation from **Az.RecoveryServices** module.

**Pull request merged**